sections/semantics-sections.include

```diff
@@ -1128,6 +1128,48 @@
         </main>
       </body>
     </xmp>
+
+    The next example shows a misuse of a <{header}>, as its content does not <a>represent</a> an
+    introduction to its parent <a>sectioning content</a>:
+
+    <xmp highlight="html">
+      <article>
+        <h2>Spinning Straw into Gold</h2>
+        <p>A primer on getting rich quick, while avoiding making horrible promises.</p>
+
+        ...
+
+        <header>
+          <h3>Make a point to learn obscure names</h3>
+        </header>
+        ...
+      </article>
+    </xmp>
+
```

Comment:
it would help to remove the ellipses, as they introduce uncertainty about what is there and so make the example harder to understand. And the extra blank lines aren't that helpful either, at least to me...